contracts/nft/src/execute/burn.rs

```diff
@@ -0,0 +1,23 @@
+use cosmwasm_std::{DepsMut, Env, MessageInfo, Response};
+use provwasm_std::types::provenance::metadata::v1::MsgDeleteScopeRequest;
+use uuid::Uuid;
+
+use crate::core::error::ContractError;
+use crate::util::action::{Action, ActionType};
+use crate::util::metadata_address::MetadataAddress;
+
+pub fn handle(
+    _deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    scope_uuid: Uuid,
+) -> Result<Response, ContractError> {
+    let burn_msg = MsgDeleteScopeRequest {
```

Comment:
Does any sort of check need to be done for who mints? I noticed you check for permissions for send.

Reply:
Just want to verify that `can_send` is correct here?

Reply:
yeah the `can_send` function verifies if it's the owner or one who has permission to handle the token like an operator